src/edu/cmu/cs/sasylf/util/ErrorHandler.java

```diff
@@ -183,4 +194,226 @@ public static Location lexicalErrorAsLocation(String file, String error) {
 			return new Location(file,0,0);
 		}
 	}
+

```

Comment:
They are declared in the `Error` enum. The reason I added methods in `ErrorHandler` is because that allows for more specific feedback to the user. Typically, error messages have the form `<error_type, specific_info>`. Each method corresponds to a particular `error_type`, and the arguments are used to construct `specific_info`.

Using these methods allows us to abstract away the logic of crafting the error message when throwing it.